### DISCUSSION TITLE
请问这个项目可以离线使用吗？

### DISCUSSION THREAD

**Discussion by michael-hongy** [Sanster/IOPaint#205] [Q&A]
请问这个项目可以离线使用吗？

**Comment by Sanster** [comment]
第一次使用某个模型必须要联网下载模型的，lama 之类的可以[手动下载](https://lama-cleaner-docs.vercel.app/install/download_model_manually)，sd 要联网下（最好加代理）

**Reply by michael-hongy** [reply]
我用的是一键安装版本，选择的是sd1. 5模型，安装后选中图片中要去除的部分没有反应，去除不了，请问是哪里的问题？
